src/main/java/cc/woverflow/hytils/mixin/GuiIngameForgeMixin_HideActionbar.java

```diff
@@ -32,6 +32,7 @@
 public class GuiIngameForgeMixin_HideActionbar {
     @Inject(method = "renderRecordOverlay", at = @At("HEAD"), cancellable = true)
     private void cancelActionBar(int width, int height, float partialTicks, CallbackInfo ci) {
-        if (HypixelUtils.INSTANCE.isHypixel() && LocrawUtil.INSTANCE.getLocrawInfo() != null && (HytilsConfig.hideHousingActionBar && LocrawUtil.INSTANCE.getLocrawInfo().getGameType() == LocrawInfo.GameType.HOUSING) || HytilsConfig.hideDropperActionBar && LocrawUtil.INSTANCE.getLocrawInfo().getGameType() == LocrawInfo.GameType.DROPPER) ci.cancel();
```

Comment:
can we update this in oneconfig too

Reply:
https://github.com/Polyfrost/OneConfig/pull/224